monomer_test.go

```diff
@@ -1,19 +1,176 @@
 package monomer_test
 
 import (
+	"math/big"
+	"strconv"
 	"testing"
+	"time"
 
+	"github.com/cometbft/cometbft/types"
+	opeth "github.com/ethereum-optimism/optimism/op-service/eth"
+	"github.com/ethereum/go-ethereum/beacon/engine"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
 	ethtypes "github.com/ethereum/go-ethereum/core/types"
+	"github.com/polymerdao/monomer"
 	"github.com/polymerdao/monomer/testutils"
 	"github.com/stretchr/testify/require"
 )
 
-func TestBlockToEth(t *testing.T) {
-	l1InfoTx, depositTx, cosmosEthTx := testutils.GenerateEthTxs(t)
-	block := testutils.GenerateBlockFromEthTxs(t, l1InfoTx, []*ethtypes.Transaction{depositTx}, []*ethtypes.Transaction{cosmosEthTx})
-	ethBlock, err := block.ToEth()
-	require.NoError(t, err)
-	for i, ethTx := range []*ethtypes.Transaction{l1InfoTx, depositTx, cosmosEthTx} {
-		require.EqualExportedValues(t, ethTx, ethBlock.Body().Transactions[i])
+func TestChainID(t *testing.T) {
+	n := int64(12345)
+	id := monomer.ChainID(n)
+
+	idStr := strconv.FormatInt(n, 10)
+	require.Equal(t, idStr, id.String())
+
+	bigInt := big.NewInt(n)
+	require.Equal(t, bigInt, id.Big())
+
+	hexID := (*hexutil.Big)(bigInt)
+	require.Equal(t, hexID, id.HexBig())
```

Comment:
The expected value should come first. `require.Equal(t, id.String(), idStr)`

Reply:
As I can see, idStr or strconv.FormatInt(n, 10) is expected

Reply:
Ahh I see, you're right.